roles/kubespray_defaults/defaults/main/main.yml

```diff
@@ -66,6 +66,15 @@ kubeadm_join_phases_skip_default: []
 kubeadm_join_phases_skip: >-
  {{ kubeadm_join_phases_skip_default }}
 
+# List of kubeadm upgrade node phases that should be skipped when upgrading a secondary control plane node (supports different phases than kubeadm init and kubeadm upgrade apply)
+kubeadm_upgrade_node_phases_skip_default: []
+kubeadm_upgrade_node_phases_skip: >-
```

Comment:
I was wondering why you choose the _default naming scheme but I see we have the same above. Best to be consistent even if it's not optimal :+1: